src/App.js

```diff
@@ -1,19 +1,28 @@
 import React from 'react';
 import './App.css';
-import Calculator from './components/Calculator';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
+import Nav from './Nav';
+import Home from './Home';
+import Quote from './Quotes';
+import Calculate from './CalculatorPg';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
-  render() {
-    return (
-      <div className="container">
-        <Calculator />
+function App() {
+  return (
+    <Router>
+      <div>
+        <Nav />
+        <Switch>
+          <Route path="/home" exact component={Home} />
```

Comment:
- [ ] Your app loads with a blank page, and this because you haven't defined a page that loads on a default path(`http://localhost:3000/`). Please is will be nice to make the home page the  default page by updating your routes as `<Route path="/" exact component={Home} />`. See image below and kindly fix that

![defaultPath](https://user-images.githubusercontent.com/65852621/145212710-b6d25f87-3cf0-44cf-bae0-82f87f7edaa5.png)
.

Reply:
Thank you